python/llm/example/pytorch-models/meta-llama/README.md

```diff
@@ -0,0 +1,94 @@
+# Llama
+
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on general pytorch models, for example Meta Llama models. For illustration purposes, we utilize the [Llama2-7b-Chat](https://ai.meta.com/llama/) as a reference Llama model.
```

Comment:
Need to make it clear:
1) This example shows how to apply BigDL-LLM optimizations to PyTorch models which are not HF 
2) It use the official LLaMA inference code from Meta (which does not use Hugging Face

Reply:
OK, I've highlighted this code source.